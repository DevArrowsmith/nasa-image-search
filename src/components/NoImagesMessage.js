import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const ErrorStyles = styled.h2`
margin: 20px 20px;
color: #D6D4D0;
`

const NoImagesMessage = () => {
  return (
    <ErrorStyles>
      {`No images found. Please try a different search term.`}
    </ErrorStyles>
  )
};

export default NoImagesMessage;