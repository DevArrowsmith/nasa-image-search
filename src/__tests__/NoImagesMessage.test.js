import React from 'react';
import { render } from '@testing-library/react';
import NoImagesMessage from '../components/NoImagesMessage';

test('component NoImagesMessage renders', () => {
  const { asFragment } = render(<NoImagesMessage />);
  const component = asFragment();
  expect(component).toMatchSnapshot();
});