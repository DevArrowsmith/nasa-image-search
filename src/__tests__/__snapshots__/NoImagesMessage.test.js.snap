// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component NoImagesMessage renders 1`] = `
<DocumentFragment>
  <h2
    class="sc-AxjAm cmwgYd"
  >
    No images found. Please try a different search term.
  </h2>
</DocumentFragment>
`;
